Dockerfile

```
@@ -23,7 +23,7 @@ ENV PATH /usr/local/go/bin:$PATH
 RUN curl -LO https://fast.dpdk.org/rel/dpdk-21.11.tar.xz && \
     echo "58660bbbe9e95abce86e47692b196555 dpdk-21.11.tar.xz" | md5sum -c && \
     tar xJf dpdk-21.11.tar.xz && cd dpdk-21.11 && \
-    meson build && cd build && ninja && ninja install
+    meson configure -Dmachine=default && meson build && cd build && ninja && ninja install
```

I've always thought that `meson configure ...` must be performed from within the build directory. Have you tested this?

```suggestion
meson build && cd build && meson configure -Dmachine=default && ninja && ninja install
```

Also, could you make it optional?
This should only be the default behavior in the CI. (There are probably some environment variables in the CI that you can check for)

---

Ou yes I think the configuration was previously wrong.

I have now also added a `CI` argument to the Dockerfile.